source/features/linkify-urls-in-code.tsx

```diff
@@ -61,17 +26,12 @@ function init(): false | void {
 			continue;
 		}
 
-		const linkified = linkifyUrls(element.textContent!, options);
-		if (linkified.children.length === 0) { // Children are <a>
-			continue;
-		}
-
-		zipTextNodes(element, linkified);
+		linkify(linkifyUrls, element, element.textContent!);
```

Comment:
zipTextNodes is probably still needed  to support linkification of URLs in diffs (see previous related PRs in the blame view) and I wonder if it should be made part of `linkify` itself (maybe not, but worth a double-check)

Reply:
I wasn't actually sure what to do with `zipTextNodes`. I had no element array to pass to it so I left it out. I thought you would bring it up if it is needed :)

What would be the best call here? As mentioned above, the previous method won't work anymore. I could add it to `linkify` though...

Reply:
Not sure, either way it probably needs to be added somewhere (haven’t tested this, but I guess it was there for that reason)

Reply:
Just checked, it basically puts HTMLElements together. Example taken from the Github page:

```js
const base = <>I live in <a href="it">Italy</a></>;
const grammar = highlightVerb(base);
// <>I <em>live</em> in Italy</> // e.g. the link was lost

zipTextNodes(base, grammar);
// <>I <em>live</em> in <a href="it">Italy</a></>
// The new `em` is copied from `grammar` to `base`
```

I guess this is really needed. I will add it to the library file so that calling `linkify` will automatically zip them.